netplan_cli/configmanager.py

```diff
@@ -153,14 +153,20 @@ def _copy_file(self, src, dst):
 
     def _copy_tree(self, src, dst, missing_ok=False):
         try:
-            shutil.copytree(src, dst)
+            shutil.copytree(src, dst, copy_function=copy_with_ownership)
         except FileNotFoundError:
             if missing_ok:
                 pass
             else:
                 raise
 
 
+def copy_with_ownership(src, dst, follow_symlinks=True):
```

Comment:
question: I wonder if we should use this function in `try_command.py:revert()` (L142), too?

Reply:
I added a commit that does it; I'm not sure that it's particularly impactful but I agree that keeping the behavior consistent is reasonable. Maybe should `s/_copy_tree/copy_tree/`.